script.js

```diff
@@ -1,12 +1,116 @@
-//You can edit ALL of the code here
+//TV Show Project
+
+const allEpisodes = getAllEpisodes();
+const episodeContainer = document.getElementById("cardContainer");
+const rootElem = document.getElementById("root");
+const searchInput = document.getElementById("searchEpisode");
+const episodesCount = document.getElementById("searchResult");
+const viewAllEpisodesButton = document.querySelector("#viewAllEpisodes");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+// display all episodes //
+
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
+  episodeList.forEach((episode) => createCard(episode));
+  episodesCount.innerText = `Displaying ${episodeList.length}/${allEpisodes.length}`;
+}
+
+// create card //
+
+function createCard(episode) {
+  let cardDiv = document.createElement("div");
+  cardDiv.classList.add("cardDiv");
+  let cardHeader = document.createElement("div");
+  cardHeader.classList.add("cardHeader");
+  let episodeTitle = document.createElement("h2");
+  let episodeSeason = document.createElement("h4");
+  let cardImg = document.createElement("img");
+  let summary = document.createElement("p");
+  episodeTitle.textContent = episode.name;
+  episodeSeason.textContent = `${formatedEpisodeNum(
+    episode.season,
+    episode.number
+  )}`;
+  cardImg.src = episode.image.medium;
+  summary.innerHTML = episode.summary;
+  cardHeader.appendChild(episodeTitle);
```

Comment:
This is one way to do it and it's not incorrect. You could have also used ` ` to write out the HTML and include the variables e.g. `season ${episodeSeason}`